### PR TITLE
Noto Sans Balinese: Version 2.004 added

### PR DESCRIPTION
**Pull request by simoncozens**
387ff1f: [gftools-packager] Noto Sans Balinese: Version 2.004 added

* Noto Sans Balinese Version 2.004 taken from the upstream repo https://github.com/notofonts/balinese.git at commit https://github.com/notofonts/balinese/commit/.

### CODE REVIEW

**Review by m4rc1e — CHANGES_REQUESTED** [review]
Collisions have been introduced when I check the text preview. 

Old:
Screenshot 2023-01-16 at 17 51 35


New:
Screenshot 2023-01-16 at 17 51 42

File: ofl/notosansbalinese/METADATA.pb [hunk 1]
```diff
@@ -23,7 +23,7 @@ axes {
 }
 source {
   repository_url: "https://github.com/notofonts/balinese.git"
-  archive_url: "https://github.com/notofonts/balinese/releases/download/NotoSansBalinese-v2.003/NotoSansBalinese-v2.003.zip"
+  archive_url: "https://github.com/notofonts/balinese/releases/download/NotoSansBalinese-v2.004/NotoSansBalinese-v2.004.zip"
 }
 is_noto: true
 languages: "ban_Bali"  # Balinese, Balinese
```
Comment (RosaWagner) [inline ofl/notosansbalinese/METADATA.pb:29]: also missing primary script

**Comment by simoncozens** [comment]
Updated

## Noto Sans Balinese: Version 2.005 added

---
 e58193b: [gftools-packager] Noto Sans Balinese: Version 2.005 added

* Noto Sans Balinese Version 2.005 taken from the upstream repo https://github.com/notofonts/balinese.git at commit https://github.com/notofonts/balinese/commit/.

**Comment by simoncozens** [comment]
```
Noto Sans Balinese v2.005

This release:
* Fixes many third-level stacking conjuncts (#3)
* Fixes several other conjunct issues (#4)
* Monospaces the musical symbols (#6)
* Improves the rendering of -u/-uu after conjuncts (#10)
* Improves the positioning of anchors on a dotted circles (#16)
* Fixes a number of overlaps (#33)
* Removes hinting from non-hinted builds (#37)
```

**Comment by vv-monsalve** [comment]
> 🔥 FAIL Please remove 'cyrillic-ext' from METADATA.pb since none of its glyphs are supported by this font file. [code: unsupported-subset]

Please confirm if we can delete these subsites.

**Comment by vv-monsalve** [comment]
- There are now some issues kerning and diacritics regresions in the latin companion
    NotoSansBalinese-Latin

- Could you please confirm about these collisions?
    
Screen Shot 2024-04-10 at 12 54 50

**Comment by simoncozens** [comment]
These are actually all fine but I have 2.006 coming out soon anyway...

**Pull request closed without merging**